.github/workflows/sync-from-sunbeam.yaml

```diff
@@ -0,0 +1,46 @@
+# This workflow is to sync dashboards and alert rules from the Sunbeam repo and create a PR if there is an update.
+name: sync-from-sunbeam
+on:
+  workflow_dispatch:
+  workflow_call:
+  schedule:
+    - cron: '43 1 * * 1'
+
+
+env:
+  REPO_OWNER: canonical
+  REPO_PATH: openstack-exporter-operator
+
+jobs:
+
+  sync-from-sunbeam:
+    name: Pull files from sunbeam
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout local repository
+        uses: actions/checkout@v4
+          
```

Comment:
Remove trailing whitespace:

```suggestion

```

If you haven't already, I'd recommend configuring your editor or IDE to display trailing whitespace.  Eg. this is what it looks like in my neovim setup:

![1722816269](https://github.com/user-attachments/assets/00e023db-680b-4d83-9bbc-8d04cd404c6c)